### ISSUE TITLE
"editor.wordWrap": true is not working in VScode 1.9.0

### ISSUE THREAD

**Issue by heyyoyo** [Microsoft/vscode#20114]
- VSCode Version: 1.9.0
- OS Version: Windows 7

Steps to Reproduce:
1- ALT+Z to toggle Word wrapping
2- I could see the property "editor.wordWrap" in my user settings
3- But not effect at all

It was working in previous version of VS code

**Comment by swemaniac** [comment]
Yup! Me too. I'm using `"editor.wrappingColumn": -1` at the moment to disable word wrapping. It's squiggly and it's not telling me why, but it works.

**Comment by mjbvz** [comment]
Closing as a duplicate of #19842

@alexandrudima's [comment](https://github.com/Microsoft/vscode/issues/19842#issuecomment-278032572) explains the new behavior

**Comment by Olanrewaju** [comment]
As at VSCode 1.10.1 
To turn on wordwrap use 
"editor.wordWrap": "on", 

Full options:
/ Controls how lines should wrap. Can be:
  //  - 'off' (disable wrapping),
  //  - 'on' (viewport wrapping),
  //  - 'wordWrapColumn' (wrap at `editor.wordWrapColumn`) or
  //  - 'bounded' (wrap at minimum of viewport and `editor.wordWrapColumn`).
  "editor.wordWrap": "off",

**Comment by eric-leitzel** [comment]
Thanks for your comment @Olanrewaju. I can see there is a lot action between the community and the developers on this issue.

I'm on VSCode 1.10.2 and I cannot turn off word wrap now. I can make it toggle for a short period of time then it seems to get stuck. Right now I have "editor.wordWrap": "off" but it is wrapping at the viewport.

Toggle Word Wrap (Alt+Z) was working for a few minutes. After going back and forth from file to file I noticed it would be turned on after I set it toggled it off. That is when I noticed the toggle no longer works. The setting for editor.wordWrap acting as if it were on, but it is set to off.

Any ideas?
